composer.json

```diff
@@ -37,7 +37,7 @@
     },
     "require-dev": {
         "doctrine/annotations": "^1.13",
-        "doctrine/coding-standard": "^9.0.2",
+        "doctrine/coding-standard": "^9.0.2 || ^10.0",
```

Comment:
We can drop 9.0 for ORM 3.0, right?

Reply:
No we can't, otherwise it's not possible to install the project on PHP 7.1 (one more reason to drop it)

Reply:
ORM 3.0 doesn't support PHP 7 anymore

Reply:
Oh right, sorry, I forgot what PR it was 👍